app/src/pages/instance/organizationsPage/organizationsPageHeader/organizationsPageHeader.jsx

```diff
@@ -65,6 +71,11 @@ export const OrganizationsPageHeader = ({
                 event={ORGANIZATION_PAGE_EVENTS.SEARCH_ORGANIZATION_FIELD}
               />
               <i className={cx('filters-icon')}>{Parser(filterIcon)}</i>
+              <FiltersFields
+                debounced={false}
+                appliedFiltersCount={appliedFiltersCount}
+                setAppliedFiltersCount={setAppliedFiltersCount}
+              />
```

Comment:
I'm getting 400 Bad request error while applying filters
Is it a known issue and where it should be fixed?
![image](https://github.com/user-attachments/assets/cae194c8-e18b-4c98-8d96-ddcfa7657b97)

Reply:
@BlazarQSO ^

Reply:
Done.
API search does not work for the conditions gte, lte for users and launches